source/reference/driver-compatibility.txt

```diff
@@ -341,6 +341,24 @@ for that Ruby version is deprecated.
      - JRuby 9.2
      - JRuby 9.1
 
+   * - 2.21
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - D
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     - |checkmark|
+     - |checkmark|
+     - D
+     -
+
```

Comment:
Q: If this is the same as the last one, can you combine them into one row 2.20 to 2.21?

Reply:
Can do